apps/frontend/src/libs/components/search/libs/hooks/use-search-query-parameter.hook.ts

```diff
@@ -0,0 +1,34 @@
+import { useLocation, useNavigate } from "react-router-dom";
+
+import { useCallback } from "~/libs/hooks/hooks.js";
```

Comment:
Why can i not grab these hooks from the same location? I am unable to do it without lint errors?